src/Microsoft.DotNet.Watcher.Tools/Internal/MsBuildProjectFinder.cs

```diff
@@ -35,20 +34,20 @@ public static string FindMsBuildProject(string searchBase, string project)
 
                 if (projects.Count > 1)
                 {
-                    throw new GracefulException(Resources.FormatError_MultipleProjectsFound(projectPath));
+                    throw new FileNotFoundException(Resources.FormatError_MultipleProjectsFound(projectPath));
```

Comment:
I was going to complain about throwing `FileNotFoundException` when in fact you've found multiple instances of what you were looking for, but I see it makes things easier for the consuming code 😄